frontend/src/app/services/api.service.ts

```diff
@@ -222,8 +222,8 @@ export class ApiService {
     );
   }
 
-  getHistoricalBlockSizesAndWeights$(interval: string | undefined) : Observable<any> {
-    return this.httpClient.get<any[]>(
```

Comment:
i think this `any[]` was copied from other API calls to here, but was incorrect. this could also be the case with other calls

Reply:
That's just me not paying enough care about typescript interfacing feature. Thanks for fixing it 👍🏻